### PR TITLE
 script for prworkflow

### PR DESCRIPTION
**Pull request by rahulraikwar00**
#756 
prworkflow
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by seema1711** [comment]
Okay, I tested this code in my local system, I have few things to add:
1. Only Active PRs are working, like this.
image
2. When I try to run, **-cpr** or **-pr**, it just displays **Done!**, but I don't get the output, instead, I get a JSON file, which I find hard to read.
3. And to make the command more flexible, we can create a new argument so that we can add **-aprv** together. 
4. And just a few spelling mistakes in the documentation.

@HarshCasper you can review it.

File: Python/prworkflow/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```shell
+$ bash install.sh
+```
+
+## usage:
+```shell
+$ python -m prworkflow <"username/repo_name"> -<argment> 
+``` 
+
+## optional arguments:
```
Comment (sohamsshah) [inline Python/prworkflow/README.md:14]: Can you add the commands in a tabular markdown form? It will be more readable.

**Review by sohamsshah — CHANGES_REQUESTED** [review]

**Comment by rahulraikwar00** [comment]
@seema1711 This is really a sample of
What would the actual project be like,
I want to make it more like a complete tool where you can give commands and no need to pass arguments
It will be like other "cli" tools, not just script.
And thank you for the feedback 👍🏻

File: Python/prworkflow/README.md (hunk 1 above)
Comment (rahulraikwar00) [reply Python/prworkflow/README.md:14]: I don't get it !😅 " Can you please suggest any example README "

**Comment by rahulraikwar00** [comment]
@vybhav72954  can you please mark this pull as draft ?

**Pull request closed without merging**